py/selenium/webdriver/common/action_chains.py

```diff
@@ -370,6 +373,69 @@ def scroll(self, x: int, y: int, delta_x: int, delta_y: int, duration: int = 0,
         )
         return self
 
+    def set_active_pointer(self, pointer_input):
+        """Sets a pointer at the time of creation of ActionChain"""
+        pointer = None
+        for device in self.w3c_actions.devices:
+            action = device.encode()
+
+            if action["id"] == pointer_input.name:
+                pointer = device
+                break
+
+        if pointer is None:
+            self.w3c_actions.pointer_action = PointerActions(pointer_input)
+        else:
+            self.w3c_actions.pointer_action = PointerActions(pointer)
+
+        return self
+
+    def set_active_keyboard(self, key_input):
+        """Sets a keyboard at the time of creation of ActionChain"""
+        keyboard = None
+        for device in self.w3c_actions.devices:
+            action = device.encode()
+
+            if action["id"] == key_input.name:
+                keyboard = device
+                break
+
+        if keyboard is None:
+            self.w3c_actions.key_action = KeyActions(key_input)
+        else:
+            self.w3c_actions.key_action = KeyActions(keyboard)
+
+        return self
+
+    def set_active_wheel(self, wheel_input):
+        """Sets a wheel at the time of creation of ActionChain"""
+        wheel = None
+        for device in self.w3c_actions.devices:
+            action = device.encode()
+
+            if action["id"] == wheel_input.name:
+                wheel = device
+                break
+
+        if wheel is None:
+            self.w3c_actions.wheel_action = WheelActions(wheel_input)
+        else:
+            self.w3c_actions.wheel_action = WheelActions(wheel)
+
+        return self
+
+    def get_active_pointer(self):
```

Comment:
This is very java-esque, can we just allow optional args to be passed in when instantiating?